gpdb-doc/dita/ref_guide/sql_commands/CREATE_TABLE.xml

```diff
@@ -343,8 +343,8 @@ CREATE [ [GLOBAL | LOCAL] {TEMPORARY | TEMP} | UNLOGGED ] TABLE [IF NOT EXISTS]
             distribution policy. Unlike <codeph>INHERITS</codeph>, the new table and original table
             are completely decoupled after creation is complete.</pd>
           <pd>
-            <note>Storage properties like append-optimized or partition structure are not
-              copied.</note>
+            <note>Partition structure is not copied. Storage properties like append-optimized are
+            only copied if <codeph>INCLUDING STORAGE</codeph> is specified.</note>
```

Comment:
I would add information that leaf partitions inherit storage settings from the root partition.

Reply:
Done